modules/010-deckhouse-flow-schema/templates/prioritylevelconfiguration.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
+kind: PriorityLevelConfiguration
+metadata:
+  name: deckhouse-prioritylevel
```

Comment:
The name is too broad and does not answer why this priority level is added.

Reply:
changed to d8-serviceaccounts